src/bluesearch/entrypoint/database/parse.py

```diff
@@ -141,24 +141,34 @@ def iter_parsers(input_type: str, input_path: Path) -> Iterator[ArticleParser]:
         raise ValueError(f"Unsupported input type '{input_type}'!")
 
 
-def run(
-    *,
-    input_type: str,
+def filter_files(
```

Comment:
Both of your comments makes sense IMO. I would also like to point out that currently the implementation lies inside of `parse.py` and then we also import this function within `topic_extract.py` which just feels wrong. Since IMO the CLI commands should be completely independent.

I will put it inside of `bluesearch.utils` and rename it. However, as @Stannislav pointed we should also be careful about not making the `utils.py` too big in the future.

Reply:
Done in 2519d6a